packages/slonik/src/dbFilters.ts

```diff
@@ -0,0 +1,114 @@
+import { QueryResultRow, TaggedTemplateLiteralInvocation, sql } from "slonik";
```

Comment:
Why not call this `filters.ts`. The `db` prefix is not required, and in any case it's an abbreviation, which we anoid.